### ISSUE TITLE
Is there any way I can keep different enabling status of this extension?

### ISSUE THREAD

**Issue by glacjay** [Croydon/vertical-tabs-reloaded#119]
Add-on version: 0.8.4.1
Firefox version: 55.0.3
OS + OS-Version: Windows 10 / macOS 10.12.6
Other tab + user interface related add-ons: 

I have two machines right now, one with a large screen, another with a little one. So I want to enable this extension on the first one, and disable it on the second.

I've enabled Firefox Sync, so I cannot do this directly within Firefox's extension manager.

**Comment by glacjay** [comment]
BTW. When I previously using Tree Style Tabs, there is an option to choose the location of the tabbar, and I chose different ones (one top / one left) on my two machines.

**Comment by grahamperrin** [comment]
I wonder. Maybe something like this combination of preferences: 

- `services.sync.prefs.sync.extensions.verticaltabsreloaded.sync.enabled`
- `services.sync.prefs.sync.extensions.verticaltabsreloaded.sync.values`

– that's loosely based on what I see for Greasemonkey (without me understanding their effects!).

----

I have what may be an overlapping use case. Sometimes I want to change the order of things in the toolbar of (legacy) Vertical Tabs Reloaded. Here, for example, I want to demote one of the two icons for _Taborama_, and bring the _List all tabs_ menu into sight:  

![2017-09-18 04 55 35](https://user-images.githubusercontent.com/192271/30530652-71659096-9c40-11e7-8174-1f5d16b6a0b4.png)

It's necessary to use **`about:addons` to disable the extension** (knowing that the change might be synced), then I reorder things in the tab bar of Firefox, finally I re-enable the extension. Result: 

![2017-09-18 04 58 01](https://user-images.githubusercontent.com/192271/30530713-0bbc39ce-9c41-11e7-908d-41d3d0c2da95.png)

– and I can reduce the sidebar without losing sight of _List all tabs_: 

![2017-09-18 05 01 35](https://user-images.githubusercontent.com/192271/30530752-545184d2-9c41-11e7-98c8-abea1d24d07e.png)

Photograph of fluffy animals = validation of use case `:-)`

Fluffy animals aside … something like that was [discussed a few weeks ago](https://matrix.to/#/!xAcrCyjpCWOtluwgSS:matrix.org/$15028381202587476LnSoN:matrix.org) but [my use case then](https://matrix.to/#/!xAcrCyjpCWOtluwgSS:matrix.org/$1502685935194430bqDyo:matrix.org) was quite obscure. 

The use case above is concrete – and I do expect to continue using Vertical Tabs Reloaded with Firefox 55.x or 56.x for months or years after 57 is released. 

## tl;dr 

Might there be a way for the extension to temporarily behave _as if_ it's disabled, without actually disabling at the `about:addons` level?

**Comment by Croydon** [comment]
This situation should be solved with the WebExtension version. For Firefox 57 it likely won't be technically possible to hide the default tab browser. With a later Firefox version it will be possible and then I will offer within VTR such an opt-out option. Meaning, in the default options Firefox's own tab browser is getting hidden, but it is possible to make it visible again.

Keeping Firefox's tab browser visible + closing VTR's tab sidebar should pretty much do it. The extension would still be enabled but shouldn't disturb you.

For the inital question: I don't know if it is possible to not sync the enabled state.

**Comment by Croydon** [comment]
@glacjay Is this problem solved for you with VTR 0.9 and Firefox 57?

You can just close the sidebar on the Firefox instance where you don't want to use it and VTR will pretty much be "disabled".

When I'm able to implement the hiding of the default tabbrowser again (#128), I will do it in an opt-out way.

**Comment by glacjay** [comment]
Yes, this solves my problem, thanks.
